core/include/mmcore/LuaInterpreter.h

```diff
@@ -203,6 +203,7 @@ std::string const megamol::core::LuaInterpreter<T>::DEFAULT_ENV =
     "  next = next,"
     "  pairs = pairs,"
     "  pcall = pcall,"
+    "  require = require,"
```

Comment:
I'm not sure I remember correctly, but didn't we want to work around `require` by using the internal file loading via `mmReadTextFile`? that way we at least _could_ sandbox that if we ever need to